client/scripts/views/pages/proposals.ts

```diff
@@ -265,11 +266,24 @@ const ProposalsPage: m.Component<{}> = {
     const visibleTechnicalCommitteeProposals = app.chain
       && (app.chain.network === ChainNetwork.Kusama || app.chain.network === ChainNetwork.Polkadot)
       && (app.chain as Substrate).technicalCommittee.store.getAll();
+    
+    const isAdmin =
+      app.user.isSiteAdmin ||
+      app.user.isAdminOfEntity({
+        chain: app.activeChainId(),
+        community: app.activeCommunityId(),
+      });
+    const isMod = app.user.isRoleOfCommunity({
+      role: 'moderator',
+      chain: app.activeChainId(),
+      community: app.activeCommunityId(),
+    });
 
     return m(Sublayout, {
       class: 'ProposalsPage',
       title: [
         'Proposals',
+        m(CommunityOptionsPopover, { isAdmin, isMod }),
```

Comment:
Instead of invoking the CommunityOptionsPopover menu every time there's a sublayout invoked, let's add it to the Sublayout component directly. If you inspect the components in the browser, you can see that they're inside the `.sublayout-header-left` div where the "Breadcrumbs" are defined (using page title). Lines 71 and 85 might be good places to invoke the carat component.